### PR TITLE
003 String Ops

### PR DESCRIPTION
**Pull request by sajir-dev**

### CODE REVIEW

File: 003_Strings.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  for (let i = 0; i < newString.length; i++) {
+    if (newString.charAt(i) == " ") {
+      newString = newString.slice(0, i) + newString.slice(i + 1, str.length);
+    }
+  }
+  return newString;
+};
+
+addcharacter = (str, chadd, ch) => {
+  for (i = 0; i < str.length; i++) {
```
Comment (kuism) [inline 003_Strings.js]: i has global scope

File: 003_Strings.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  return rev;
+};
+
+reverseconcat = (str1, str2) => {
+  return reversestring(stringconcat(str1, str2));
+};
+
+/*
+stringmix = (str1, str2)=>{
```
Comment (kuism) [inline 003_Strings.js]: why is this not implemented

File: 003_Strings.js (hunk 1 above)
Comment (sajir-dev) [reply 003_Strings.js]: will correct now

File: 003_Strings.js (hunk 2 above)
Comment (sajir-dev) [reply 003_Strings.js]: because I wasn't getting output here

Comment (sajir-dev) [reply 003_Strings.js]: I will settle this issue anyways

File: 003_Strings.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+};
+
+reverseconcat = (str1, str2) => {
+  return reversestring(stringconcat(str1, str2));
+};
+
+
+stringmix = (str1, str2)=>{
+	let mixstring = ""
+	for (let i = 0; i<(str1+str2).length; i++){
```
Comment (kuism) [inline 003_Strings.js]: alignment not proper

File: 003_Strings.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+		if (i%2 == 0){
+			mixstring = mixstring+str1.charAt(i/2)
+		}else{
```
Comment (kuism) [inline 003_Strings.js]: allignment

File: 003_Strings.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
+stringconcat = (str1, str2) => str1 + " " + str2;
+
+reversestring = (str) => {
+  rev = "";
```
Comment (kuism) [inline 003_Strings.js]: rev is not declared

Comment (sajir-dev) [reply 003_Strings.js]: rev declared

**Pull request merged**